toolbox/fdc3-explained/README.md

```diff
@@ -1,4 +1,4 @@
-# <a href='http://fdc3.finos.org/toolbox/fdc3-explained'><img src='logo.png' alt='FDC3 Explained Logo'></a>
+# <a href='http://fdc3.finos.org/toolbox/fdc3-explained'><img src='logo.png' alt='FDC3 Explained Logo'/>
```

Comment:
We lost having the image within the link.  Should this be changed to markdown instead of html?

Reply:
This is just an error that needs reverting I think. The A tag should definitely contain the image still. We may need to look for other cases that need reverting.

Well spotted @bingenito!

The equivalent markdown (for interest's sake) is:

```
[![FDC3 Explained Logo](logo.png)](http://fdc3.finos.org/toolbox/fdc3-explained)
```